ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -40,70 +40,77 @@ This ACP does not introduce these, but some form of asynchronous execution is re
-Similarly, it MUST record the transaction-receipt root for all transactions executed as part of chunks in the half-open range $t_C \in (t_{B-1} - d, t_B - d]$.
-It is at this point that a transaction is said to be _settled_ on the blockchain.
+There is a constantly alive block executor whose job is to execute blocks taken off the FIFO execution queue.
 
-#### Terminology recap
+In addition to executing the blocks, the block executor provides deterministic timestamps for the beginning and end of execution of a block.
 
-In summary, a transaction is _included_ when added to a _built_ block.
-It is later _accepted_ after consensus agrees upon the block in which it was included, enqueuing the transaction (as part of the block's _tranche_) for later _execution_ in a chunk.
-The chunk is _full_ when it exhausts either the queue or its gas limit, and the transaction is _settled_ when its receipt is part of a later block agreed upon by consensus.
-While the delay between execution and settlement is always $d$ seconds, the time between acceptance and execution is variable.
-Note, however, that the validity checks we now describe mean that acceptance is a guarantee of eventual settlement in a deterministic but as yet unknown block.
+Time is accordingly measured in two ways by the block executor.
 
-```mermaid
-flowchart LR
-    I[Included] --> A[Accepted]
-    A -->|variable delay| E[Executed]
-    E -->|d seconds| S[Settled]
-    A -. guarantees .-> S
-```
+1. The timestamp included in the block header.
+2. The amount of gas charged during the execution of blocks.
 
 > [!NOTE]
-> The guarantee provides no assurance about whether or not a transaction will revert nor whether its computation will run out of gas by reaching the specified limit.
-> However, this is not a concern for EOA-to-EOA transfers of value so such transactions are guaranteed in totality.
+> Execution timestamps are more granular than block header timestamps to allow sub-second block execution times.
+
+As soon as there is a block available to execute on the execution queue, the block executor starts processing the block.
+
+If the block executor's current timestamp is prior to the current block's timestamp, the block executor's timestamp is advanced to the block's timestamp.
+
+The block is then executed on top of the last executed (not settled) state.
+
+After executing the block, the block executor advances its timestamp based on the gas usage of the block.
+
+The block's execution time is now timestamped and the block is available to be settled.
+
+#### Settling blocks
+
+Previously executed blocks are settled by the acceptance of a new block whose timestamp is greater than or equal to the execution time of a prior block plus a constant additional delay.
 
-### Updating gas price
+The additional delay amortises any spurious slowdowns the block executor may have encountered.
```

Comment:
```suggestion
The additional delay amortizes any sporadic slowdowns the block executor may have encountered.
```

Reply:
The whole ACP currently uses the "ise" rather than "ize" suffix. Either is fine (English vs American) but it should be consistent imo

Reply:
-ise 🇦🇺